python/feathub/processors/flink/table_builder/flink_table_builder.py

```diff
@@ -234,16 +234,26 @@ def _get_table_from_derived_feature_view(
             Tuple[str, Sequence[str]], Dict[str, JoinFieldDescriptor]
         ] = {}
 
+        # This list contains all ExpressionTransform features listed after the first
+        # OverWindowTransform or JoinTransform feature in the dependent_features.
```

Comment:
Maybe change to `JoinTransform or OverWindowTransform` for a bit more consistency with the idea that we evaluate `JoinTransform` features before `OverWindowTransform` features.